### DISCUSSION TITLE
skip checkup failed for Android

### DISCUSSION THREAD

**Discussion by aulimge** [skiptools/skip#57] [Q&A] [answered]
<img width="655" alt="Screenshot 2023-11-22 at 11 29 05 PM" src="https://github.com/skiptools/skip/assets/30146547/4a2f8a05-f550-45ba-9d20-27e231f28dc4">

and for xcode i have this error
![Screenshot 2023-11-22 at 11 29 29 PM](https://github.com/skiptools/skip/assets/30146547/380e9dff-18b5-437b-b167-e583408b30e7)

**Comment by marcprux** [comment]
`gradle` should have been installed as a side-effect of `brew install skiptools/skip/skip`. What happens when you run `gradle -version`? Can you try `brew reinstall gradle` and see if it changes anything?

**Reply by aulimge** [reply]
>brew reinstall gradle
Error: Cannot install in Homebrew on ARM processor in Intel default prefix (/usr/local)!
Please create a new installation in /opt/homebrew using one of the
"Alternative Installs" from:
  https://docs.brew.sh/Installation
You can migrate your previously installed formula list with:
  brew bundle dump

**Reply by marcprux** [reply, marked as answer]
It sounds like you are using an ARM mac and have an old homebrew installation from a previous Intel machine. 

The best bet might be to remove the Intel installation (https://docs.brew.sh/FAQ#how-do-i-uninstall-homebrew) and re-install Homebrew from scratch.

**Reply by aulimge** [reply]
I have done the Homebrew uninstall and install back
brew update done
Do i need to reinstall skip again ?

**Reply by marcprux** [reply]
You might need to re-install Skip as well. You can do it with brew reinstall skiptools/skip/skip, and the hopefully skip checkup will pass.

**Reply by aulimge** [reply]
yes now it works , can run HelloSkip 
Question do i need to remove any old skip folder which was install in Intel processor since now i reinstall into ARM processor (Apple Silicon M1) ?

**Comment by aulimge** [comment]
![Screenshot 2023-11-24 at 10 48 54 PM](https://github.com/skiptools/skip/assets/30146547/fe455351-094e-48cc-9348-604aae524aee)
it works now....

Now trying to convert existing app and see how it works

**Comment by roslog00** [comment]
Hi, I'm having trouble running tests. These errors occur, I saw solutions in discussions but didn't understand how to apply them. Please advise, has anyone managed to resolve these issues?
<img width="441" alt="Снимок экрана 2023-12-08 в 11 37 41 AM" src="https://github.com/skiptools/skip/assets/135430018/6485d569-9373-4efb-abd0-b2ee206d0928">
<img width="521" alt="Снимок экрана 2023-12-08 в 11 38 34 AM" src="https://github.com/skiptools/skip/assets/135430018/9d463a58-0cf4-4ade-a6b4-e87cc45878d1">
